### ISSUE TITLE
Bold and italics rendering in Markdown

### ISSUE THREAD

**Issue by sergiorgiraldo** [silvia-odwyer/xenon#1]
When writing bold and italics, the preview does not recognize markup for bold and italic. Instead, it renders a list with * at the end.

Steps to reproduce:
1. Start with a blank note
2. Write *foo. It renders a bullet with string "foo"
3. Write * (now the string is \*foo\*). It renders a bullet with string "foo*"

Expected behaviour: renders _foo_

Same goes with bold markup (2 *)

**Comment by silvia-odwyer** [comment]
Hi Sergio, 
Thank you for raising this issue, I fixed this in the latest two commits. 
Italics and bold text should now render correctly, and lists are now started with a hyphen rather than an asterisk.
So the markup to create a list would be as follows:

```
- item 1
- item 2
- item 3
```

Bold and italics can also be used within lists, so there should be no problems there either.

After investigating, I saw that this bug existed, was because I unintentionally changed the ordering of the regex which parsed the lists before the bold/italics, leading to lists being created rather than formatting the text.

Let me know if you have any other bugs you'd like to report, and hopefully this solves the issue! Thanks again.
